crates/primitives/starknet/src/tests/crypto.rs

```diff
@@ -121,10 +120,40 @@ fn test_merkle_tree() {
     let event_com = calculate_event_commitment::<PedersenHasher>(&events);
     // The values we test ours against are computed from the sequencer test.
     assert_eq!(
-        H256::from_str("0x03ebee479332edbeecca7dee501cb507c69d51e0df116d28ae84cd2671dfef02").unwrap(),
+        Felt252Wrapper::from_hex_be("0x03ebee479332edbeecca7dee501cb507c69d51e0df116d28ae84cd2671dfef02").unwrap(),
         event_com
     );
-    assert_eq!(H256::from_str("0x054c0fddf3aaf1ca03271712b323822647b66042ccc418ba1d7fb852aebfd2da").unwrap(), tx_com);
+    assert_eq!(
+        Felt252Wrapper::from_hex_be("0x054c0fddf3aaf1ca03271712b323822647b66042ccc418ba1d7fb852aebfd2da").unwrap(),
+        tx_com
+    );
+}
+
+#[test]
+fn test_merkle_tree_class_commitment() {
+    let class_hashes = vec![Felt252Wrapper::from(0_u128), Felt252Wrapper::from(1_u128)];
+
+    let class_com = calculate_class_commitment_tree_root_hash::<PedersenHasher>(&class_hashes);
+
+    // The values we test ours against are computed from the sequencer test.
```

Comment:
is this true ?

Reply:
Actually wanted to ask what did you mean when writing this ?

Reply:
I went in the Cairo lang repo and took the values from the python tests. I don’t think those tests are external though and I don’t think we test specifically the state root